processor/src/main/java/org/mapstruct/ap/internal/model/Mapper.java

```diff
@@ -31,6 +31,8 @@
 import org.mapstruct.ap.internal.option.Options;
 import org.mapstruct.ap.internal.version.VersionInformation;
 
+import static org.mapstruct.Mapper.DEFAULT_IMPL_NAME;
```

Comment:
As @agudian is saying there should be no compile dependency to the annotation module.